tests/test__default.py

```diff
@@ -235,6 +235,15 @@ def test__get_gae_credentials(app_identity):
     assert project_id == mock.sentinel.project
 
 
+def test__get_gae_credentials_no_app_engine():
+    import sys
+    with mock.patch.dict('sys.modules'):
```

Comment:
Thanks! Any idea why Travis is still failing the coverage check? https://travis-ci.org/googleapis/google-auth-library-python/jobs/469551157

Reply:
`.tox/cover/lib/python3.6/site-packages/google/auth/_default.py                       104      2     26      0    98%   179-180` so it thinks it isn't testing an import error being thrown from `_get_gae_credentials`

It does seem to me this tests should be covering that line.

Reply:
OK. Should I submit this CL? My concern is that Travis will continue to fail for future PRs if I do.